ansible/roles/jenkins-worker/vars/main.yml

```diff
@@ -91,6 +91,7 @@ needs_monit: [
 
 # some os'es needs different paths to java. add them here.
 java_path: {
+  'centos7': '/home/iojs/jdk8u172-b11/bin/java',
```

Comment:
This java path is actually specific to centos-ppc64le, but I'm not sure if any other centos7 platforms use systemd.